modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,83 @@
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements in this release.
 
+[#release-664]
+== Release 6.6.4 (December 2021)
+
+Couchbase Server 6.6.4 was released in December 2021. This maintenance release includes bug fixes in multiple components.
+
+[#fixed-issues-664]
+=== Fixed Issues
+
+This section highlights the notable issues fixed in this release.
+
+==== Cross Data Center Replication (XDCR)
+
+[#table_fixedissues_v664-xdcr,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-48212[MB-48212^]
+| *Summary*: File descriptor leak in XDCR
+
+| https://issues.couchbase.com/browse/MB-48134[MB-48134^]
+| *Summary*: Remove staged transaction xattr and replicate data
+
+| https://issues.couchbase.com/browse/MB-48801[MB-48801^]
+| *Summary*: Compressed binary documents with xattrs are not replicated
+
+|===
+
+[#cluster-manager]
+==== Cluster Manager
+
+[#table_fixedissues_v664-cluster-manager,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-47613[MB-47613^]
+| *Summary*: Add bucket names and UUIDs to pools/default
+
+| https://issues.couchbase.com/browse/MB-49271[MB-49271^]
+| *Summary*: ns_server should generate DCP connection names that are no longer than allowed by memcached (relates to xref:release-notes:relnotes.adoc#data-service[MB-34280])
+|===
+
+==== Eventing Service
+
+[#table_fixedissues_v664-eventing-service,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-48289[MB-48289^]
+| *Summary*: Investigate eventing-producer RSS bloat when eventing-consumers are re-spawned continuously 
```

Comment:
Perhaps drop the word "Investigate" or change it to "Fix" the current text reads like we did anything.

Reply:
I will fix in the 6.6.5 release notes, which this content now becomes.